dubbo-bom/pom.xml

```diff
@@ -342,6 +342,11 @@
                 <artifactId>dubbo-serialization-gson</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.dubbo</groupId>
+                <artifactId>dubbo-serialization-googlePb</artifactId>
```

Comment:
rename to 'dubbo-serialization-protobuf-json'

Reply:
OK